crates/subspace-node/src/chain_spec.rs

```diff
@@ -303,6 +308,7 @@ pub fn dev_config() -> Result<ConsensusChainSpec<GenesisConfig>, String> {
                     enable_rewards: false,
                     enable_storage_access: false,
                     allow_authoring_by: AllowAuthoringBy::Anyone,
+                    pot_slot_iterations: NonZeroU32::new(150_000_000).expect("Not zero; qed"),
```

Comment:
why higher iterations for dev? Make sense to have same for both local and dev. 
Maybe use this iterations for devnet instead ?

Reply:
Hm... I meant to have it lower here and higher in devnet, nice catch!